zion/src/main/scala/edu/uci/ics/cloudberry/zion/asterix/TwitterViewsManagerActor.scala

```diff
@@ -18,7 +20,7 @@ class TwitterViewsManagerActor(val conn: AsterixConnection, override val sourceA
 
   override def getViewKey(query: DBQuery): String = {
     val keyword = query.predicates.find(_.isInstanceOf[KeywordPredicate]).map(_.asInstanceOf[KeywordPredicate].keywords.head).getOrElse("")
-    sourceName + "_" + keyword
+    sourceName + "_" + MessageDigest.getInstance("MD5").digest(keyword.getBytes("UTF-8")).map("%02x" format _).mkString
```

Comment:
[`MD5`](https://en.wikipedia.org/wiki/MD5) is a hash function to generate a unique checksum for a given bytes array. We are using it here to differentiate different keywords. Previously we were directly using `keyword` itself, but it won't work for the non-ASCII UTF-8 character. Because we are using this `key` to generate AQL dataset names. By using the HEX string of `MD5` value will work in AQL.